api/v1/errorHandler.js

```diff
@@ -54,6 +55,17 @@ function isDbError(err) {
 }
 
 /**
+ * Indicates whether the error is a Cache error as defined by our cacheErrors
+ * module.
+ *
+ * @param {Error} err - The error to test
+ * @returns {Boolean} true if err is defined as one of our Cache errors
+ */
+function isCacheError(err) {
+  return err instanceof cacheErrors.UpdateDeleteForbidden;
```

Comment:
We missed it for other errors and it didnt come up in reviews as well. Good that you noticed. It would take only few additional lines to add. I can do it for other errors in my next PR.